driver-core/src/test/java/com/datastax/driver/core/CCMAccess.java

```diff
@@ -236,4 +243,14 @@
      */
     void waitForDown(int node);
 
+    /**
+     * @return The desired target protocol version based on the 'cassandra.version' System property.
+     */
+    ProtocolVersion getDesiredProtocolVersion();
```

Comment:
Suggestion: replace "desired" – its meaning is quite unclear here – with "target", e.g. `getTargetProtocolVersion` os even simply `getProtocolVersion`.
The javadocs need to be updated, here the returned version is the version that matches this specific instance, not the global system property.